examples/.scripts/README-text.md

```diff
@@ -0,0 +1,41 @@
+This directory contains examples of protocol specifications written in Quint.
```

Comment:
I was confused why this is here. I think other people might stumble upon this file (e.g., via search) and have the same problem.

Can we somewhere note that this is a template, at least in the filename?